webcrypto/crypto.go

```diff
@@ -1,16 +1,94 @@
 package webcrypto
 
-import "go.k6.io/k6/js/modules"
+import (
+	"crypto/rand"
+	"errors"
+	"fmt"
 
+	"github.com/dop251/goja"
+	"go.k6.io/k6/js/common"
+	"go.k6.io/k6/js/modules"
+)
+
+// Crypto represents the Crypto interface of the Web Crypto API.
 type Crypto struct {
 	vu modules.VU
 
 	Subtle    *SubtleCrypto      `json:"subtle"`
 	CryptoKey *CryptoKey[[]byte] `json:"CryptoKey"`
 }
 
-func (c *Crypto) GetRandomValues() {
-	// TODO
+// QuotaExceededError is the error thrown if the byteLength of a typedArray
+// exceeds 65,536.
+var QuotaExceededError = errors.New("QuotaExceededError")
+
+// TypeMismatchError is the error thrown if the typedArray given to
+// getRandomValues is not of a supported type.
+var TypeMismatchError = errors.New("TypeMismatchError")
+
+// GetRandomValues lets you get cryptographically strong random values.
+//
+// Do not generate keys using the getRandomValues method. Use the generateKey method instead.
+//
+// The array given as the parameter is filled with random numbers (random in
+// its cryptographic sense, not in its statistical sense).
+//
+// To guarantee enough performance, this implementation is not using a truly
+// random number generator, but is using a pseudo-random number generator
+// seeded with a value with enough entropy. We are using the golang
+// crypto/rand package, which uses the operating system's random number
+// generator.
+func (c *Crypto) GetRandomValues(typedArray goja.Value) goja.Value {
+	// 1.
+	// If the provided typedArray is not of an integer type (Int8Array, Uint8Array,
```

Comment:
I appreciate your comments (here and in many other places), but do you think we need them when the code explains itself? 

I mean, the lines below bellow are easy to understand and don't require any additional commenting. Comments have their cost, e.g., maintenance; they quickly become obsolete if somebody forgets to update them during some one-line fix. 

Generally, we have less text (code + comments) in the method, which is more accessible from the cognitive side to get the point.

Reply:
You're right, actually I meant to replace the comment with a `var ()` statement with all the checks to make the `if` statement clearer, will do just that 👍🏻